src/chocolatey/infrastructure.app/nuget/NugetCommon.cs

```diff
@@ -58,6 +58,7 @@ public static IPackageRepository GetRemoteRepository(ChocolateyConfiguration con
 
             // ensure credentials can be grabbed from configuration
             HttpClient.DefaultCredentialProvider = new ChocolateyNugetCredentialProvider(configuration);
+            HttpClient.DefaultCertificateProvider = new ChocolateyClientCertificateProvider(configuration);

```

Comment:
Unless I'm missing something, this is setting a value EQUAL to the second, and not adding to a list. What i think is needed here is either a list OR a custom meta type that incorporates both of them (this latter bit I think is the right answer).